clients/include/rocblas_test_unique_ptr.hpp

```diff
@@ -1,52 +1,52 @@
-#ifndef GUARD_ROCBLAS_MANAGE_PTR

```

Comment:
The changes in this file are cosmetic: Removal of tab characters and trailing whitespace, and the use of the "do {} while (0)" idiom to make function-like macros syntactically equivalent to function calls.